rs/sns/governance/src/governance.rs

```diff
@@ -4727,7 +4728,16 @@ impl Governance {
         // The following code must remain after the async call.
         if let Some(ref mut cached_upgrade_steps) = self.proto.cached_upgrade_steps {
```

Comment:
```suggestion
        let cached_upgrade_steps = todo!();
        self.proto.cached_upgrade_steps.replace(cached_upgrade_steps);
```

Reply:
Think if this is a more prominent way to express the fact that we are definitely refreshing this.

Reply:
Ok, I updated it and imo it's much more clear now